Update K8s app installation instructions (#3295)
SumoLogic/sumologic-documentation: docs/integrations/containers-orchestration/kubernetes.md

The Deployment Guide has information on advanced configurations, best practices, performance, troubleshooting, and upgrading for our latest and previous versions of supported software.

## Installing the Kubernetes app

Now that you have set up the collection for Kubernetes app, install the Sumo Logic App for Kubernetes to use the pre-configured Kubernetes dashboards that provide visibility into your Kubernetes environment.

To install the app, do the following:

1. Locate and install the app from the **App Catalog**.
2. Search for **Kubernetes** and select the app.
3. Optionally, on this page, you can also see a preview of the dashboards included with the app.
4. Click **Add Integration**.

> 5. On the next configuration page, titled **Install App**, follow the instructions (Add Repo, Monitor & Alert Installation, Install Helm chart). When you're done, click **Next**.

Author: ```suggestion
5. On the next configuration page, titled **Install App**, follow the instructions (Add Repo, Monitor & Alert Installation, Install Helm chart). When you're done, click **Next**.
   :::note 
   Helm chart installation via command line is required in order to install the Kubernetes app properly and see the dashboards in the next step. 
   :::
```

> 6. (Optional) On the next configuration step, titled **Preview & Done**, click the blue **Explore Dashboards** button or **Open Dashboards in Library** to check out your Kubernetes dashboards. Here, you'll also be given the options to add another cluster, configure Kubernetes monitors, or manage your collectors.

Reviewer: May be we can add a warning that if you click on next without finishing the helm chart commands then Open Dashboards in library link won't work.

Author: To clarify, are you saying that you need to finish the Helm chart commands, or else the data won't load into the dashboards at all?

Reviewer 2: I tried out the onboarding & found out that Helm command is responsible for the creation of the dashboards. Basically, app doesn't get installed if that command is not run. Since, the intention of user is to install the app and have dashboards, this command is a must. @himanshu219 correct me if I am wrong.

Reviewer: yes that's correct

Reviewer 3: The `helm` is mandatory step. Without it the app won't be installed.

Author: does this sound ok? _Helm chart installation via command line is required in order to install the Kubernetes app properly and see the dashboards in the next step._

Reviewer 2: We can remove "properly". Rest wordings looks good to me. Make sure we add this to the instructions for the "Install App" step.

The Deployment Guide has information on advanced configurations, best practices, performance, troubleshooting, and upgrading for our latest and previous versions of supported software.

## Installing the Kubernetes app

Now that you have set up the collection for Kubernetes app, install the Sumo Logic App for Kubernetes to use the pre-configured Kubernetes dashboards that provide visibility into your Kubernetes environment.

To install the app, do the following:

1. Locate and install the app from the **App Catalog**.
2. Search for **Kubernetes** and select the app.
3. Optionally, on this page, you can also see a preview of the dashboards included with the app.
4. Click **Add Integration**.
5. On the next configuration page, titled **Install App**, follow the instructions (Add Repo, Monitor & Alert Installation, Install Helm chart). When you're done, click **Next**.
   :::note 

>    Helm chart installation via command line is required in order to install the Kubernetes app properly and see the dashboards in the next step. 

Author: ```suggestion
   Helm chart installation via command line is required in order to install the Kubernetes app and see the dashboards in the next step. 
```

   1. Run `terraform plan` to view the monitors which will be created/modified by Terraform.
   1. Run `terraform apply`.
7. **Post Installation**. If you haven’t enabled alerts and/or configured notifications through the Terraform procedure outlined above, we highly recommend enabling alerts of interest and configuring each enabled alert to send notifications to other people or services. See Step 4 of [this document](/docs/alerts/monitors#add-a-monitor).

:::note
There are limits to how many alerts can be enabled - see the [Alerts FAQ](/docs/alerts).
:::

## Upgrading the Kubernetes app

The current version of our app is not backwards compatible with older Helm Chart versions. Follow the below steps to upgrade.

1. Switch to [Admin Mode](/docs/manage/content-sharing/admin-mode/#switch-to-admin-mode). If you are not a Content Administrator, you may have to contact Administrator within your organization to grant the [Manage Content](/docs/manage/users-roles/roles/role-capabilities/#data-management) role to your user.

> 1. Go to `Library/Admin Recommended/Sumo Logic Integrations/Kubernetes` and rename the folder (for example, Kubernetes-backup-Oct-2023) by clicking on the three button and choosing **Edit**.

Reviewer 2: Let's add a note that after user renames the folder, it may take around 5min for changes to take effect. So wait for 5min & proceed to the next step after that.

Author: @hamoli can you please review?

```suggestion
1. Go to the folder **Library/Admin Recommended/Sumo Logic Integrations/Kubernetes**, click on the three button, choose **Edit**, and rename the folder to your liking (for example, **Kubernetes-backup-Nov-2023**). The renaming may take up to 5 minutes to go into effect.
```

Reviewer 2: LGTM

> 1. Go back to the **App Catalog** and reinstall the Kubernetes app again by following the [Installing the Kubernetes app](/docs/integrations/containers-orchestration/kubernetes/#installing-the-kubernetes-app) instructions. If the tab was already open, you may have to refresh the page.

Reviewer 3: I think that the switch to `Content Administrator` is only required to rename the folder. After renaming the folder user can go back to `Me` mode and install `Kubernetes` app.

> 1. Go back to the **App Catalog** and reinstall the Kubernetes app again by following the [Installing the Kubernetes app](/docs/integrations/containers-orchestration/kubernetes/#installing-the-kubernetes-app) instructions. If the tab was already open, you may have to refresh the page.

Author: Does this sound ok?

```suggestion
1. If you're in **Content Administrator** mode, switch back to **Me** mode.
1. Go back to the **App Catalog** and reinstall the Kubernetes app again by following the [Installing the Kubernetes app](/docs/integrations/containers-orchestration/kubernetes/#installing-the-kubernetes-app) instructions. If the tab was already open, you may have to refresh the page.
```

> 1. Go back to the **App Catalog** and reinstall the Kubernetes app again by following the [Installing the Kubernetes app](/docs/integrations/containers-orchestration/kubernetes/#installing-the-kubernetes-app) instructions. If the tab was already open, you may have to refresh the page.

Reviewer: LGTM

> 1. Go back to the **App Catalog** and reinstall the Kubernetes app again by following the [Installing the Kubernetes app](/docs/integrations/containers-orchestration/kubernetes/#installing-the-kubernetes-app) instructions. If the tab was already open, you may have to refresh the page.

Reviewer 3: Looks good 👍
